src/default/assets/js/src/typedoc/components/Toggle.ts

```diff
@@ -3,25 +3,22 @@ namespace typedoc
     /**
      * Enabled simple toggle buttons.
      */
-    class Toggle extends Backbone.View<any>
-    {
+    class Toggle extends Backbone.View<any> {
         active?: boolean;
 
         className!: string;
 
-
-        constructor(options:Backbone.ViewOptions<any>) {
+        constructor(options: Backbone.ViewOptions<any>) {
             super(options);
 
-            this.className = this.$el.attr('data-toggle');
-            this.$el.on(pointerUp, (e) => this.onPointerUp(e));
-            this.$el.on('click', (e) => e.preventDefault());
-            $document.on(pointerDown, (e) => this.onDocumentPointerDown(e));
-            $document.on(pointerUp, (e) => this.onDocumentPointerUp(e));
+            this.className = this.$el.attr('data-toggle') || '';
+            this.$el.on(pointerUp, e => this.onPointerUp(e));
```

Comment:
I generally prefer the explicit `(e)` parenthesis since they are needed most of the time and make it clear what the params are but you don't need to change it back.

Reply:
Fair point, changed for consistency.